client/libra-dev/Cargo.toml

```diff
@@ -24,7 +24,7 @@ transaction-builder = { path = "../../language/transaction-builder", version = "
 rand = "0.7.3"
 
 [build-dependencies]
-bindgen = "0.54.1"
```

Comment:
Just want to confirm this is intended.

Reply:
yes it is, need to downgrade, upgrade is blocked by conflicts on clang-sys 1.0.0/0.23.0